src/LogInUser.java

```diff
@@ -0,0 +1,35 @@
+import Exceptions.UserException;
+
+import java.util.Scanner;
+
+public class LogInUser {
+
+    public void validation(Scanner scanner, User registeredUser) {
```

Comment:
Add an input object for login operation

Reply:
still not input object

Reply:
create a separate input object for example RegisterUser that will contain the necessary info and then convert it to User and save it